Wikipedia/Code/WMFSettingsViewController.m

```diff
@@ -103,7 +103,7 @@ - (void)viewWillAppear:(BOOL)animated {
 
 - (void)configureNotificationsButton {
     if ([[NSUserDefaults standardUserDefaults] defaultTabType] == WMFAppDefaultTabTypeSettings && [[[self dataStore] authenticationManager] isLoggedIn]) {
-        UIBarButtonItem *notificationsBarButton = [[UIBarButtonItem alloc] initWithImage:[UIImage imageNamed:@"notifications-bell"] style:UIBarButtonItemStylePlain target:self action:nil];
+        UIBarButtonItem *notificationsBarButton = [[UIBarButtonItem alloc] initWithImage:[UIImage imageNamed:@"notifications-bell"] style:UIBarButtonItemStylePlain target:self action:@selector(userDidTapNotificationsCenter)];
```

Comment:
I haven't looked into it, but for some reason this button doesn't show on Settings when I launch the app after turning the Explore feed off.

Reply:
Let's cover this one in https://phabricator.wikimedia.org/T288669 – that seems like the best opportunity to be sure all our bell display logic is solid.